docs/sites.yml

```diff
@@ -4203,3 +4203,16 @@
   built_by: Maciej Leszczyński
   built_by_url: http://twitter.com/_maciej
   featured: false
+- title: Frame.io Workflow Guide
+  main_url: https://workflow.frame.io
+  url: https://workflow.frame.io
+  description: >
+    The web’s most comprehensive post-production resource, written by pro filmmakers, for pro filmmakers. Always expanding, always free.
+  # You can list as many categories as you want here. Check list of Categories below in this doc!
```

Comment:
Want to remove these two comments? Otherwise this looks great!

Reply:
@DSchau Duh, removing now!